hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/JdbcSqlTestSupport.java

```diff
@@ -101,11 +101,11 @@ protected String quote(String... parts) {
      * Creates table with id INT, name VARCHAR columns
      */
     public static void createTable(String tableName) throws SQLException {
-        createTable(tableName, "id INT PRIMARY KEY", "name VARCHAR(100)");
+        createTable(tableName, "\"id\" INT PRIMARY KEY", "\"name\" VARCHAR(100)");
     }
 
     public static void createTable(String tableName, String... columns) throws SQLException {
-        executeJdbc("CREATE TABLE " + tableName + " (" + String.join(", ", columns) + ")");
```

Comment:
You will see that this method is called from many places, maybe we could have a variant that quotes the first token from each column automatically.

Reply:
Could you check out this solution for the executeJdbc variant https://github.com/hazelcast/hazelcast/pull/25255/commits/88db4c2fde960862cebf0a2abe57994a5edf9742
If it is what you want, I will change the function calls in the tests accordingly.